src/plugins/poster/poster.js

```diff
@@ -108,6 +108,12 @@ export default class PosterPlugin extends UIContainerPlugin {
   }
 
   update() {
+    let hideOnPlay = true;
+    if (this.options.posterConfig && typeof(this.options.posterConfig.hideOnPlay) != 'undefined') {
+      // There are circumstances to keep poster around when we'd normally hide it
+      // (e.g. audio-only source inside video container format)
+      hideOnPlay = this.options.posterConfig.hideOnPlay;
+    }
```

Comment:
This can be simplified to:

let hideOnPlay = !!(this.options.posterConfig && this.options.posterConfig.hideOnPlay);